Fixed issue #2328 positioning of social media buttons in wiki page

sukhbir-singh opened the pull request:
Fixed Issue #2328 
Make sure these boxes are checked before your pull request is ready to be reviewed and merged. 

* [x] all tests pass -- `rake test:all`
* [x] code is in uniquely-named feature branch, and has been rebased on top of latest master (especially if you've been asked to make additional changes)
* [x] pull request is descriptively named with #number reference back to original issue
Thanks!

Screenshot of solved positioning issue:-

![screenshot from 2018-02-15 02-02-34](https://user-images.githubusercontent.com/13816752/36228060-14d86c8e-11f9-11e8-8b74-53f042916412.png)

sukhbir-singh commented:
@Gauravano I have updated the commit as requested.

Gauravano commented:
@sukhbir-singh I guess there is a change in view.
![seethis](https://user-images.githubusercontent.com/20878070/36266110-c0e18068-1296-11e8-9ec9-519e6788921b.png)

sukhbir-singh commented:
@Gauravano Please review this again. I have added vertical-align:middle to making its appearance same on firefox and chrome both. Last time i have checked only on chrome. Fixed it now. :)

Gauravano commented on app/views/home/_social.html.erb:
```diff
@@ -1,7 +1,7 @@
-<div id="social">
+<div id="social" style="position: absolute">
```
style can be removed

sukhbir-singh replied:
I this it will break the functioning of "position:relative" of facebook and twitter buttons.

Gauravano replied:
try it once

Gauravano commented on app/views/home/_social.html.erb:
```diff
@@ -1,7 +1,7 @@
   <% if Rails.env == "production" %>
-  <div class="fb-like" data-send="false" data-layout="button_count" data-width="50" data-show-faces="false" style="vertical-align:top;zoom:1;display:inline"></div>
-  <a style="padding-left:3px;" href="https://twitter.com/share" class="twitter-share-button">Tweet</a>
+  <div class="fb-like" data-send="false" data-layout="button_count" data-width="50" data-show-faces="false" style="position: relative;display: inline-block;vertical-align: middle; bottom: 6px;margin-right:10px" ></div>
```
position and display can be removed

Gauravano commented on app/views/home/_social.html.erb:
```diff
@@ -1,7 +1,7 @@
-  <div class="fb-like" data-send="false" data-layout="button_count" data-width="50" data-show-faces="false" style="vertical-align:top;zoom:1;display:inline"></div>
-  <a style="padding-left:3px;" href="https://twitter.com/share" class="twitter-share-button">Tweet</a>
+  <div class="fb-like" data-send="false" data-layout="button_count" data-width="50" data-show-faces="false" style="position: relative;display: inline-block;vertical-align: middle; bottom: 6px;margin-right:10px" ></div>
+  <a style="position: relative;width: 60px;height: 20px;display: inline-block;bottom: 7px;" href="https://twitter.com/share" class="twitter-share-button">Tweet</a>
```
display and position can be removed

sukhbir-singh commented:
Thank you @Gauravano for correcting me.

jywarren commented:
Indeed -- fantastic. Thank you!!!
